uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/LeaveRequest.php

```diff
@@ -753,4 +754,29 @@ private static function getLeaveRequestDayTypeOptionsGroupByValue() {
     }
     return $leaveRequestDayTypeOptionsGroup;
   }
+
+  /**
+   * This method overrides the addSelectWhereClause method of the Core DAO Class.
+   * Generates acl clauses suitable for adding to WHERE or ON when doing an api.get for this entity
+   *
+   * Return format is in the form of fieldname => clauses starting with an operator. e.g.:
+   * @code
+   *   [
+   *     'contact_id' => 'IN (1,2,3)'
+   *   ]
+   * @endcode
+   *
+   * @return array
```

Comment:
Since this is just duplicating the  doc from the DAO class, you can just use @inheritDoc